### PR TITLE
add new company -  Brainstorm Force

### PR DESCRIPTION
**Pull request by nilanshsharma**
This is a modified version of, and should adhere to, the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

* [ ] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply**
* [ ] You are an employee of the company mentioned and confirm all included details are correct.
* [x] You know your alphabet - company is listed in alphabetical order in the README.md file.
* [x] The company directly hires employees. ** No bootcamps / freelance sites / etc**
* [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
* [x] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
* [x] An entry has been added or edited within the [readme.md file](../../main/README.md).
* [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
* [x] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
* [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

**Review by KieranRobson — APPROVED** [review]
Looks good to me!

File: company-profiles/brainstorm-force.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Our open roles include both in-person and remote options across countries, offices and teams
+
+## Region
+
+India
+
+## Company technologies
+
+Wordpress, QA Engineer, Seo, Angular, React, Copywriter, Ui Designer
```
Comment (dougaitken) [inline company-profiles/brainstorm-force.md:21]: WordPress - capital P.

**Review by dougaitken — CHANGES_REQUESTED** [review]

**Comment by nilanshsharma** [comment]
Hey @dougaitken I've made the change as you asked for.

**Comment by dougaitken** [comment]
Thanks @nilanshsharma 

One final one I noticed, the region is India but another sentence hints are more globally distributed employees. Can you confirm and clarify that for me, please?

**Comment by nilanshsharma** [comment]
Hey @dougaitken, I confirmed with the company and updated the status.

File: company-profiles/brainstorm-force.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+51-200+ employees (Source: [LinkedIn](https://www.linkedin.com/company/brainstorm-force/))
+
+## Remote status
+
+Our open roles include both in-person and remote options across India, offices and teams.
+
+## Region
+
+India
```
Comment (dougaitken) [inline company-profiles/brainstorm-force.md:17]: Change this to Worldwide if that's correct please @nilanshsharma

**Review by dougaitken — APPROVED** [review]

**Comment by nilanshsharma** [comment]
Hey @dougaitken all changes confirmed and updated as you asked for.

**Review by dougaitken — APPROVED** [review]

**Comment by dougaitken** [comment]
Hey there, thanks for contributing, it is appreciated

**Pull request merged**